src/views/PeopleSearch/components/SearchFieldList/index.tsx

```diff
@@ -521,28 +524,66 @@ const SearchFieldList = ({ onSearch }: Props) => {
                       select
                     />
                   ) : (
-                    <Grid item width={225}>
-                      <Slider
-                        getAriaLabel={() => 'graduationYearRange'}
-                        value={graduationYearRange}
-                        onChange={handleSliderChange}
-                        valueLabelDisplay="auto"
-                        getAriaValueText={toString}
-                        min={1889}
-                        max={currentYear}
-                        disabled={!searchParams.includeAlumni}
-                        color="secondary"
-                      />
-                      <Typography fontSize={15} align="center">
-                        {graduationYearRange[0]}-{graduationYearRange[1]}
-                      </Typography>
+                    <Grid item container spacing={1}>
+                      <Grid item xs={6} md={6.75}>
+                        <SearchField
+                          name="initial_year"
+                          value={searchParams.initial_year}
+                          updateValue={handleUpdate}
+                          options={Array.from(
+                            {
+                              length:
+                                Number(searchParams.final_year) > 0
+                                  ? Number(searchParams.final_year) - 1889 + 1
+                                  : currentYear - 1889 + 1,
+                            },
+                            (_, i) => ({
+                              value:
+                                Number(searchParams.final_year) > 0
+                                  ? (Number(searchParams.final_year) - i).toString()
+                                  : (currentYear - i).toString(),
+                              label:
+                                Number(searchParams.final_year) > 0
+                                  ? (Number(searchParams.final_year) - i).toString()
+                                  : (currentYear - i).toString(),
+                            }),
+                          ).reverse()}
+                          Icon={FaCalendarTimes}
+                          disabled={!searchParams.includeAlumni}
+                          defaultDisabled
+                          select
+                        />
+                      </Grid>
+                      <Grid item xs={6} md={5.25}>
+                        <SearchField
+                          name="final_year"
+                          value={searchParams.final_year}
+                          updateValue={handleUpdate}
+                          options={Array.from(
+                            {
+                              length:
+                                Number(searchParams.initial_year) > 0
+                                  ? currentYear - Number(searchParams.initial_year) + 1
+                                  : currentYear - 1889 + 1,
+                            },
```

Comment:
```suggestion
                          options={Array.from(
                            {
                              length:
                                Number(searchParams.initial_year) > 0
                                  ? currentYear - Number(searchParams.initial_year) + 1
                                  : currentYear - 1889 + 1,
                            },
```
The fact that the exact same code for length exists twice makes me think there can be an auxiliary function to do this?